improver/utilities/cube_manipulation.py

```diff
@@ -96,6 +96,21 @@ def collapse_realizations(cube: Cube) -> Cube:
     return returned_cube
 
 
+def collapse_percentiles(cube: Cube) -> Cube:
```

Comment:
The, very similar, `collapse_realizations` function in this file has corresponding unit tests in `test_collapse_realizations.py`, should unit tests be added for this new function too? 

I also think that, due to the similarity of the `collapse_realizations` and `collapse_percentiles` functions, they could probably be combined into a single function. Is the decision not to do this to avoid making lots of changes to e.g a new function name, or new required inputs?

Reply:
I've added unit tests for the `collapse_percentiles` function. As discussed, although `collapse_realizations` and `collapse_percentiles` are very similar, it is not obvious how best to combine them given the function signature expected in the `BasicThreshold` plugin, so I've left these functions unchanged.